speech/api/speech_async_rest.py

```diff
@@ -57,8 +57,12 @@ def main(speech_file):
     service_request = service.speech().asyncrecognize(
         body={
             'config': {
-                'encoding': 'LINEAR16',
-                'sampleRate': 16000
+                # There are a bunch of config options you can specify. See
+                # https://cloud.google.com/speech/reference/rest/Shared.Types/RecognitionConfig  # noqa
```

Comment:
I'm generally not a fan of our tools causing us to add things into sample code that wouldn't otherwise be there. (Principle of least surprise).

We should either:
1) leave the line.
2) use a shorturl.